config/default/manager_auth_proxy_patch.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     spec:
       containers:
       - name: kube-rbac-proxy
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0
+        image: quay.io/brancz/kube-rbac-proxy:v0.10.0 
```

Comment:
I'm not sure about this one. I trust the kubebuilder build process; I don't know who `quay.io/brancz` is or if we should trust their images. I'd prefer something like `make manifests-multi` so that people have to explicitly opt into using this version of kube-rbac-proxy from an untrusted source. Or even just tell people to modify the manifest themselves (e.g. change `gcr.io/k8s-staging-multitenancy/hnc-manager:v0.9.0` to `gcr.io/k8s-staging-multitenancy/hnc-manager:v0.9.0-multi`, and change `gcr.io/kubebuilder/kube-rbac-proxy:v0.4.0` to `quay.io/brancz/kube-rbac-proxy:v0.10.0` _at your own risk_).

Is there a bug against kubebuilder to produce multiarch images?

Reply:
I think the ownership has been passed and KubeBuilder maintainers are contributing to the same repo.  Here's the [repo ](https://github.com/brancz/kube-rbac-proxy) and here's a comment by one of the maintainers suggesting to use the image(https://github.com/kubernetes-sigs/kubebuilder/issues/1637#issuecomment-746035030) .

Reply:
Ah, ok, I see that `github.com/brancz` is the source-of-truth for kube-rbac-proxy. I'm just not sure about their supply chain. Let me ask and find out why kubebuilder thought it was necessary to have their own copies in the first place.

Reply:
So I asked around, and apparently `gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` is multi-architecture. I think I'd prefer to use that instead if it works.

The reason it's in `gcr.io/kubebuilder` is that this registry is owned by a Kubernetes SIG, while `quay.io` is not. This isn't an absolutely massive deal but it's why I'd prefer to follow kubebuilder's lead and use that image if it's reasonable to.